start.R

```diff
@@ -67,18 +67,20 @@ stoppedWithError <- tryCatch({
   return(TRUE)
 })
 
+today <- format(Sys.time(), "%Y-%m-%d")
```

Comment:
OK, does not throw an error anymore.  But now it reports "APT done" for every run, not just APTs.

Reply:
I am really not in good shape anymore.